### PR TITLE
Update trust-policies chart with support for trust domain

### PR DESCRIPTION
**Pull request by codysoyland**
Fixes https://github.com/github/package-security/issues/2105

Signed-off-by: Cody Soyland <codysoyland@github.com>

### CODE REVIEW

**Review by ejahnGithub — APPROVED** [review]
lgtm

File: charts/trust-policies/values.yaml
Comment (steiza) [inline charts/trust-policies/values.yaml:31]: Minor: I don't see any mention of trust domains on this documentation today - maybe it's getting updated later?

File: charts/trust-policies/Chart.yaml
Comment (steiza) [inline charts/trust-policies/Chart.yaml:11]: Minor / just to make sure I understand: we're making this tag after this pull request lands?

**Review by steiza — APPROVED** [review]
Minor non-blocking Q's, LGTM!

File: charts/trust-policies/values.yaml
Comment (phillmv) [reply charts/trust-policies/values.yaml:31]: @steiza separate PR

File: charts/trust-policies/values.yaml
Comment (phillmv) [inline charts/trust-policies/values.yaml:30]:
```suggestion
    # $ gh api meta -q .domains.artifact_attestations.trust_domain
```

Comment (phillmv) [reply charts/trust-policies/values.yaml:30]: (turns out gh ships with jq?)

File: charts/trust-policies/Chart.yaml
Comment (codysoyland) [reply charts/trust-policies/Chart.yaml:11]: That's right!

File: charts/trust-policies/values.yaml
Comment (codysoyland) [reply charts/trust-policies/values.yaml]: Good call! I pushed a new commit using the long form flag `--jq`.

**Review by steiza — APPROVED** [review]
The setting that keeps dismissing reviews is quite annoying...

**Pull request merged**
